activerecord/lib/active_record/persistence.rb

```diff
@@ -621,8 +621,14 @@ def reload(options = nil)
 
     # Saves the record with the updated_at/on attributes set to the current time
     # or the time specified.
-    # Please note that no validation is performed and only the +after_touch+,
-    # +after_commit+ and +after_rollback+ callbacks are executed.
+    #
+    # * No validation is performed.
+    # * Only the +after_touch+, +after_commit+ and +after_rollback+ callbacks
+    #   are executed.
+    # * The dirty-tracking state is cleared, so unpersisted changes are no
+    #   longer tracked by methods like `changed?`, `previous_changes`,
```

Comment:
The backticks can't use for code font in RDoc.  
Please change to use `+` just like the previous line.